tests/test_command_reference.rb

```diff
@@ -45,13 +45,15 @@ def write(string)
 
   def test_data_sanity
     # Make sure the actual YAML in our library loads for various platforms
-    CommandReference.new('', '')
-    CommandReference.new('NXAPI', '')
-    CommandReference.new('NXAPI', 'N9K-C9396PX')
-    CommandReference.new('NXAPI', 'N7K-C7010')
-    CommandReference.new('NXAPI', 'N3K-C3064PQ-10GE')
-    CommandReference.new('gRPC', '')
-    CommandReference.new('gRPC', 'R-IOSXRV9000-CH')
+    CommandReference.new
+    CommandReference.new(platform: :nexus, cli: true)
```

Comment:
I think I would prefer we default cli: to true instead of false.  Future cases that don't support cli: can do the explicit set.